packages/core/src/helpers/calls.ts

```diff
@@ -20,29 +20,37 @@ export function warnOnInvalidCall(call: Call | Falsy) {
 /**
  * @internal Intended for internal use - use it on your own risk
  */
-export function encodeCallData(call: Call | Falsy, chainId: number, queryParams: QueryParams = {}): RawCall | Falsy {
+export function validateCall(call: Call | Falsy) {
   if (!call) {
-    return undefined
+    throw new Error('Call is falsy')
```

Comment:
Make sure the behaviour is unchanged.

Add a test for something like:
```
const result = useCall(undefined)
expect(result).to.xxx
```

And make sure the behaviour is the same before and after your changes.